patches/components-content_settings-core-browser-content_settings_utils.cc.patch

```diff
@@ -1,19 +1,72 @@
 diff --git a/components/content_settings/core/browser/content_settings_utils.cc b/components/content_settings/core/browser/content_settings_utils.cc
-index a66ca0ac3f01cc6a92c54bbaaef031b7e6414510..259023be6103322c999fe781e1033c8f1f66c75e 100644
+index a66ca0ac3f01cc6a92c54bbaaef031b7e6414510..af095d036fbf05a575e86bb1da00bc7eaf040d0e 100644
 --- a/components/content_settings/core/browser/content_settings_utils.cc
 +++ b/components/content_settings/core/browser/content_settings_utils.cc
-@@ -148,6 +148,14 @@ void GetRendererContentSettingRules(const HostContentSettingsMap* map,
+@@ -43,12 +43,9 @@ static_assert(base::size(kContentSettingsStringMapping) ==
+ // belong between ALLOW and ASK. DEFAULT should never be used and is therefore
+ // not part of this array.
+ const ContentSetting kContentSettingOrder[] = {
+-    CONTENT_SETTING_ALLOW,
+-    CONTENT_SETTING_SESSION_ONLY,
+-    CONTENT_SETTING_DETECT_IMPORTANT_CONTENT,
+-    CONTENT_SETTING_ASK,
+-    CONTENT_SETTING_BLOCK
+-};
++    CONTENT_SETTING_ALLOW, CONTENT_SETTING_SESSION_ONLY,
```

Comment:
@pilgrim-brave this looks like a formatting change. Guessing it just snuck in by accident.